src/server/game/Entities/Player/Player.cpp

```diff
@@ -17293,9 +17289,8 @@ bool Player::LoadFromDB(ObjectGuid guid, CharacterDatabaseQueryHolder const& hol
 
 #define RelocateToHomebind()                                                                                                                                                                           \
```

Comment:
This define can be swapped for an auto function. Cleans up the code a bit.
```cpp
auto RelocateToHomebind = [this, &instanceId]()
{
    instanceId = 0;
    Relocate(m_homebind);
};
```

Reply:
Thank you
https://github.com/FirelandsProject/firelands-cata/pull/164/commits/9c04febf3a83122bea2941c1bceab85249442e11